jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -328,6 +339,22 @@ frontend http-in
   <%- if properties.ha_proxy.frontend_config -%>
     <%= p("ha_proxy.frontend_config") %>
   <%- end -%>
+  <%- if_p("ha_proxy.connections_rate_limit.table_size", "ha_proxy.connections_rate_limit.window_size") do -%>
```

Comment:
Currently, we're performing *rate-limiting, allowlisting, blocklisting* in exactly this order.
This has a disadvantage:
If a landscape is allowlisted or blocklisted, the stick-tables will still fill up for clients that are not on the allowlist (or on the blocklist) theoretically resulting in more of a performance hit on haproxy.
We can't put rate-limiting behind allowlisting, because once a request is accepted with `tcp-request content accept`, we cannot reject it any more.
The other possible solution would be *blocklist, rate-limit, allowlist*, but that would change behaviour on landscapes that are both allowlisted and blocklisted.